tests/integration/__snapshots__/index.test.js.snap

```diff
@@ -1624,6 +1624,134 @@ exports[`integration: produce a BOM that includes hashes from package-lock.json
         }
       ]
     }
+  ],
+  \\"dependencies\\": [
```

Comment:
a tree needs a root. i do not see the root component.
could you help me and point me to it. @kabo ? 

there must be a `ref` to the component that is set in `bom.tools.component.bom-ref`

Reply:
Root component added.